app/locale/en.coffee

```diff
@@ -709,7 +709,10 @@
     support_part2: "support@codecombat.com"
     support_part3: "if you have any questions."
 
-  announcement:
+  announcement: 
+    now_available: "Now available for subscribers!"
+    subscriber: "subscriber"
+    # Pet Announcement Modal
```

Comment:
Put these comments as trailing comments following the first line in the section; don't add a comment-only line (the scripts won't handle it)